app/helpers/wpcc/your_results_helper.rb

```diff
@@ -0,0 +1,14 @@
+module Wpcc
+  module YourResultsHelper
+    TAX_RELIEF_THRESHOLD_RATE = {
```

Comment:
This constant will be refactored as part of the improvement work on constants. See [TP#8446](https://moneyadviceservice.tpondemand.com/entity/8446) for details on this upcoming work on how to store constants